backend/cmd/server.go

```diff
@@ -125,9 +127,16 @@ func GetContextKeyAndKContext(w http.ResponseWriter,
 
 // CacheMiddleWare is Middleware for Caching purpose. It involves generating key for a request,
 // authorizing user , store resource data in cache and returns data if key is present.
-func CacheMiddleWare(c *HeadlampConfig) mux.MiddlewareFunc {
+func CacheMiddleWare(c *HeadlampConfig) mux.MiddlewareFunc { //nolint:funlen, gocognit
```

Comment:
Can you please remove these lint disable lines?

if you move some of the added stuff into their own functions, then it should be easier to test, and also make disabling the lint rules unnecessary.